Sources/ButtonBarViewCell.swift

```diff
@@ -44,9 +43,9 @@ open class ButtonBarViewCell: UICollectionViewCell {
         set {
             super.isSelected = newValue
             if (newValue) {
-                accessibilityTraits |= UIAccessibilityTraitSelected
+                accessibilityTraits = [.selected]
             } else {
-                accessibilityTraits &= ~UIAccessibilityTraitSelected
+                accessibilityTraits.formUnion(.selected)
```

Comment:
And here you should remove selected and not add it: `accessibilityTraits.remove(.selected)`